patches/popt/aarch64_support.patch

```diff
@@ -0,0 +1,733 @@
+#-------------------------------------------------------------------------------
+#
+#  Copyright (C) 2016 Pankaj Bansal <pankajbansal3073@gmail.com>
+#
+#  SPDX-License-Identifier:     GPL-2.0
```

Comment:
This looks great.

The stg patch comment should indicate where this patch came from.  I'm assuming you back ported this patch from somewhere.